ZuluContent/Skills/ItemIdentification.cs

```diff
@@ -39,7 +39,7 @@ protected override void OnTarget(Mobile from, object o)
                         if (!item.Identified)
                             item.Identified = true;
                         item.OnSingleClick(from);
-                        from.SendAsciiMessage(MessageSuccessHue, "It appears to be " + GetMagicItemName(item));
+                        from.SendSuccessMessage("It appears to be " + GetMagicItemName(item));
```

Comment:
Ah damn coulda used an $ string here, guess this is from my old code

Reply:
Yeah there's no perf difference but it's nicer as an interpolated string; there should also be a Cliloc entry for this I think